go-controller/pkg/node/vrfmanager/vrf_manager.go

```diff
@@ -298,8 +298,8 @@ func (vrfm *Controller) repair(validVRFs sets.Set[string]) error {
 			if err != nil {
 				klog.Errorf("VRF Manager: error deleting stale VRF device %s, err: %v", name, err)
 			}
+			delete(vrfm.vrfs, link.Attrs().Index)
```

Comment:
nit: yeah the other option was to continue if it was validVRFs.Has(name); neater for reading loop